test/verify/check-superuser

```diff
@@ -438,13 +437,26 @@ class TestSuperuserOldWebserver(testlib.MachineCase):
         b.click('#login-button')
         b.wait_visible('#content')
 
-        # We should have gotten the password from the old cockpit-ws and it should work
-        b.check_superuser_indicator("Administrative access")
-        b.go("/playground/test")
-        b.enter_page("/playground/test")
-        b.click(".super-channel button")
-        b.wait_in_text(".super-channel span", 'result: ')
-        self.assertIn('result: uid=0', b.text(".super-channel span"))
+        if not self.is_pybridge():
+            # The C bridge will recognize that it is being started by
+            # a old webserver (or old bridge) and will start "any"
+            # privileged bridge during startup.
+            b.check_superuser_indicator("Administrative access")
+            b.go("/playground/test")
+            b.enter_page("/playground/test")
+            b.click(".super-channel button")
+            b.wait_in_text(".super-channel span", 'result: ')
+            self.assertIn('result: uid=0', b.text(".super-channel span"))
+
+        else:
+            # The Python bridge will not start any privileged bridge
+            # if not explicitly told, and a old webserver (or old
+            # bridge) does not tell it anything.
+            b.check_superuser_indicator("Limited access")
+            b.go("/playground/test")
+            b.enter_page("/playground/test")
+            b.click(".super-channel button")
+            b.wait_in_text(".super-channel span", 'access-denied')
```

Comment:
Can you please add a check here that clicking on the priv indicator can actually get you root? I think that's the most important part of it.

Reply:
Yes, good point.